.travis.yml

```diff
@@ -5,17 +5,17 @@ language: python
 
 env:
   matrix:
-    - PYTHON_VERSION="2.6" NUMPY_VERSION="1.6"
-    - PYTHON_VERSION="2.7" NUMPY_VERSION="1.7"
-    - PYTHON_VERSION="3.3" NUMPY_VERSION="1.8"
-    - PYTHON_VERSION="3.4" NUMPY_VERSION="1.9"
+    - PYTHON_VERSION="2.6" NUMPY_VERSION="1.6" PYTEST_VERSION="2.6.3"
```

Comment:
I see that you have reverted your changes, but don't worry about Python 2.6 we are going to drop support for it pretty soon.